cssize.hpp

```diff
@@ -0,0 +1,88 @@
+#ifndef SASS_CSSIZE
+#define SASS_CSSIZE
+
+#include <vector>
+#include <iostream>
+
+#ifndef SASS_AST
+#include "ast.hpp"
+#endif
+
+#ifndef SASS_OPERATION
+#include "operation.hpp"
+#endif
+
+#ifndef SASS_ENVIRONMENT
+#include "environment.hpp"
+#endif
+
+namespace Sass {
+  using namespace std;
+
+  struct Context;
+  class Eval;
+  class Contextualize;
+  class Expand;
+  typedef Environment<AST_Node*> Env;
+  struct Backtrace;
+
+  class Cssize : public Operation_CRTP<Statement*, Cssize> {
+
+    Context&            ctx;
+    Env*                env;
+    vector<Block*>      block_stack;
+    vector<Statement*>  p_stack;
+
+    Statement* fallback_impl(AST_Node* n);
```

Comment:
I'm not sure why this cause compilation to fail on only some platform though. More this file is just a modified version of [eval.hpp](https://github.com/sass/libsass/blob/master/eval.hpp)

Which declares this as [protected](https://github.com/sass/libsass/blob/master/eval.hpp#L30).